examples/camunda-8.7-irsa/opensearch.tf

```diff
@@ -0,0 +1,131 @@
+locals {
+  opensearch_domain_name = "domain-name-os-irsa" # Replace "domain-name" with your domain name
+
+  opensearch_master_username = "secret_user"    # Replace with your opensearch username
+  opensearch_master_password = "Secretvalue$23" # Replace with your opensearch password
```

Comment:
Should we mention the pw requirements or link to something upstream?

Ran into this as my pw didn't fit the requirements.

> The master user password must contain at least one uppercase letter, one lowercase letter, one number, and one special character.